### DISCUSSION TITLE
How can I set cockpit.mydomain.com

### DISCUSSION THREAD

**Discussion by Ludo-code** [cockpit-project/cockpit#17354] [Q&A]
Hello i have followed all our instruction in doc but doesn't working so if anyone know how to set custom domain on cockpit this can be help me so much. Thanks in advance for our reply :) 
tried with apache virtual host and njinx proxy...

**Comment by jelly** [comment]
Hey, have you tried to following the nginx wiki page? https://github.com/cockpit-project/cockpit/wiki/Proxying-Cockpit-over-nginx

**Reply by Ludo-code** [reply]
Yes i have already tried it out but continue to redirect on my other webserver on port 443

**Reply by Ludo-code** [reply]
I don't have tell it in my first message but I want to use certbot to automatically update certificate, idk if this avec one importance...
